src/Apps/Notifications/NotificationsApp.tsx

```diff
@@ -1,12 +1,18 @@
+import { Box } from "@artsy/palette"
 import { MetaTags } from "Components/MetaTags"
-import { NotificationsListQueryRenderer } from "Components/Notifications/NotificationsList"
+import { Notifications } from "Components/Notifications/Notifications"
 
 export const NotificationsApp: React.FC = () => {
   return (
     <>
       <MetaTags title="Notifications | Artsy" pathname="/notifications" />
 
-      <NotificationsListQueryRenderer />
+      <Box
+        // Unconventional value here to keep visual rhythm
```

Comment:
Generally we use `4` — what about this requires a 3?

Reply:
the offset of 30px is specified in the design 😅

<img width="599" alt="image" src="https://user-images.githubusercontent.com/3513494/187238703-d6bb44ce-0db5-4f65-9547-b574082121d8.png">

Reply:
but I agree with you that it is better to make 40px so that there is a correspondence to other places

<img width="1040" alt="image" src="https://user-images.githubusercontent.com/3513494/187239386-dd90986c-6fd1-42ad-8f23-69335bc66500.png">

Reply:
but I agree with you that it is better to make 40px so that there is a correspondence to other places

Reply:
@dimatretyak - The design is incorrect: we have no "30" space in our design system. We need to take it back to design.

Reply:
I just left a comment for Ryan in the Figma spec [here](https://www.figma.com/file/SlzmAJV6zqJDmAO8GoSjiC?node-id=1336:82114#257544863).

I see that the spacing was changed to `4` (=40px). This UI isn't live yet (it's behind a feature flag). We'll make sure that the implementation is aligned with design (and that the design is aligned with the design system), well before launching anything.

Reply:
Cool 👍 its more just a matter of developers knowing that we can go back to design and mention the inconsistency versus just creating whatever is send to us. Sometimes it's just an accident on the part of the designer, or a new designer not familiar with the constraints of the design sys (and so on).